src/aiida/orm/nodes/data/code/installed.py

```diff
@@ -147,6 +154,13 @@ def validate_filepath_executable(self):
                 f'The provided remote absolute path `{self.filepath_executable}` does not exist on the computer.'
             )
 
+        if not user_has_execute:
+            execute_msg = (
+                f'The executable at the remote absolute path `{self.filepath_executable}` exists, '
```

Comment:
```suggestion
                f'File `{self.filepath_executable}` exists, '
```
Let's called it a file if we're not sure that's an executable :)